### ISSUE TITLE
getting data in a single mbtiles file

### ISSUE THREAD

**Issue by Swanson-Hysell** [rclark/grand-canyon-geology#28]
Hi Ryan,

Is there a way to download a .mbtiles file with all of these data (or to generate it)? We are headed to the canyon and having the .mbtiles file to use in Fieldmove would be great.

Thanks,

Nick

**Comment by rclark** [comment]
Hi @Swanson-Hysell! There is not an mbtiles file. I manage these data in a couple of big GeoJSON documents. There are tools out there that can help load GeoJSON into mbtiles, but I haven't tried to maintain that. If you want, you can download those files [at the URLs listed here](https://github.com/rclark/grand-canyon-geology#getting-the-data).

Are you heading into the canyon really soon? I might be able to put something together if we have time. But usually, there's some confusion between how to _ship and access the data_ and _how to visualize it_ (i.e. colors and symbols). It is done in a particular way for the website I maintain that won't translate to mbtiles. Sorting that out would probably take some work between the both of us.

**Comment by Swanson-Hysell** [comment]
Thanks for the quick response. I have accessed the GeoJSON files. These days I have been making mbtiles using QGIS. I can do that using your GeoJSON, but as you say I would need to redo all of the styling from scratch. It was because of this (and how great your styling is) that I reached out wondering whether there was a more straightforward way to generate an mbtiles file that incorporates your styling of the GeoJSON. QGIS also struggles to render such a large GeoJSON.

I am headed in to the canyon soon — week from today. So there isn't too much time to get it together.
